users/dierkes/preprocessing/wav2vec.py

```diff
@@ -0,0 +1,69 @@
+import soundfile
+import glob
+import os
+import random
+
+from sisyphus import Job, Task, tk
+
+class FairseqWav2VecManifestCreationJob(Job):
+    """
+    Creates required manifest files for wav2vec pretraining with fairseq
+
+
+    """
+    def __init__(self, audio_path, file_extension='wav', valid_percent=0.01, seed=42, 
+            path_must_contain=None):
+        """
+
+        :param tk.Path audio_path: path to raw audio files to be included
```

Comment:
I asssume this is supposed to be a directory? Then you could make this clear, in the variable name and the description.